mod_import_anymeta.erl

```diff
@@ -434,7 +437,10 @@ import_thing(Host, AnymetaId, Thing, Stats, Context) ->
                         {anymeta_rsc_uri, RscUri},
                         {anymeta_host, Host},
                         {language, Langs},
-                        {is_authoritative, Authoritative}
+                        {is_authoritative, Authoritative},
+                        {rights, Rights},
```

Comment:
This contains the copyright (actually: creative commons). Shall we name it copyrights (or creative_commons)?

/cc @arjan